locations/spiders/burger_king.py

```diff
@@ -16,10 +16,12 @@ class BurgerKingSpider(scrapy.Spider):
 
     name = "burgerking"
     allowed_domains = ["locations.bk.com"]
-    download_delay = 0
+    download_delay = 1.5
     start_urls = (
         'https://locations.bk.com/index.html',
     )
+    def handle_error(self, failure):
+        self.log("Request failed: %s" % failure.request)
```

Comment:
There's no need to add a error handler. The existing one will log the error already.